### ISSUE TITLE
installation instructions can use some additions

### ISSUE THREAD

**Issue by Xantios** [hooklift/node-libvirt#52]
When going through the initial instructions on a debian setup you need some additional dependency's which are not stated in the instructions maybe add something like:

You need the libvirt-dev on debian stable ( apt-get install libvirt-dev ) and maybe the libicu-dev package ( apt-get install libicu-dev )

After download these packages go to the cloned repo and run "npm install nan" 

now you can continue building using the gyp command ( node-gyp configure )

**Comment by mbroadst** [comment]
@Xantios you're right, the instructions _are_ out of date. Specifically they should just be:
```
$ sudo apt-get install libvirt-dev
$ git clone git://github.com/hooklift/node-libvirt.git
$ cd node-libvirt
$ npm install
$ node-gyp configure
$ node-gyp build
```

Or something to that effect. Would you be interested in cleaning up the docs and submitting a PR? Contributions are more than welcome

**Comment by mbroadst** [comment]
fixed in 220294a0932df940568681a5b0b32507715afb92
